questionnaire/routes.js

```diff
@@ -74,6 +77,45 @@ router.route('/start').get(async (req, res) => {
     }
 });
 
+router.get('/resume/:questionnaireId', async (req, res) => {
+    try {
+        const accountService = createAccountService(req.session);
+        const questionnaireService = createQuestionnaireService({
+            ownerId: accountService.getOwnerId(),
+            isAuthenticated: accountService.isAuthenticated(req)
+        });
+
+        const defaultRedirect = '/apply';
+        let redirectUrl = defaultRedirect;
```

Comment:
Just initialise this with the default rather than creating a new const, i.e. `let redirectUrl = '/apply';`